kivy_ios/toolchain.py

```diff
@@ -360,9 +360,6 @@ def __init__(self):
 
         # path to some tools
         self.ccache = sh.which("ccache")
-        if not self.ccache:
-            # ccache is missing, the build will not be optimized
-            pass
```

Comment:
I also sneaked another quick fix. This was basically almost like a "no operation" so I dropped it